packages/app-desktop/gui/NoteEditor/utils/contextMenu.ts

```diff
@@ -92,6 +93,15 @@ export function menuItems(): ContextMenuItems {
 			},
 			isActive: (itemType: ContextMenuItemType) => itemType === ContextMenuItemType.Image || itemType === ContextMenuItemType.Resource,
 		},
+		copyImage: {
+			label: _('Copy Image'),
+			onAction: async (options: ContextMenuOptions) => {
+				const { resourcePath } = await resourceInfo(options);
+				const image = nativeImage.createFromPath(toSystemSlashes(resourcePath));
```

Comment:
what would happen if it's not an image?

Reply:
Or if it's an image that's encrypted

Reply:
Copy Image option will only show if the item is of type image or resource. So, if it is other than that, it will not show.

Reply:
Yes. So should I show this option only for an image?

Reply:
Well, `nativeImage` won't for for non-image resources for sure.
Should we allow copying attachments other than images is a different question. Not sure if any other app allows it.

Reply:
> Well, `nativeImage` won't for for non-image resources for sure.

Yes, it works only for image resource.

> Should we allow copying attachments other than images is a different question. Not sure if any other app allows it.

So, in that case I will push another commit with restricting it to only image type.

Reply:
> So, in that case I will push another commit with restricting it to only image type.

This makes sense, yes. Also please check what happens for encrypted images.